lib/Semantics/expression.cpp

```diff
@@ -2502,7 +2503,7 @@ bool ExpressionAnalyzer::EnforceTypeConstraint(parser::CharBlock at,
     const MaybeExpr &result, TypeCategory category, bool defaultKind) {
   if (result) {
     if (auto type{result->GetType()}) {
-      if (type->category() != category) { // C885
+      if (type->category() != category) {  // C885
```

Comment:
additional space before comment?

Reply:
It was added by clang-format-7 for some reason.

Reply:
I did not run clang-format in this file as it adds the extra space.